jinja_utils.py

```diff
@@ -82,23 +81,6 @@ def get_jinja_env(dir_path):
         os.path.dirname(__file__), dir_path))
     env = jinja2.Environment(autoescape=True, loader=loader)
 
-    def get_complete_static_resource_url(domain_url, resource_suffix):
-        """Returns the relative path for the resource, appending it to the
-        corresponding cache slug.
-
-        Args:
-            domain_url: str. The url of the domain.
-            resource_suffix: str. The resource suffix to get the relative path
-                for the resource. It should have a leading slash.
-
-        Returns:
-            str. The relative path for the resource.
-        """
-        return '%s%s%s' % (
-            domain_url, utils.get_asset_dir_prefix(), resource_suffix)
-
-    env.globals['get_complete_static_resource_url'] = (
```

Comment:
Is this intentional?

Reply:
Yes. 
`get_complete_static_resource_url` is no longer used.